src/actors/rqgActorSheet.ts

```diff
@@ -240,7 +240,7 @@ export class RqgActorSheet extends ActorSheet<
       }
 
       // the tens value of the percentage of the value compared to the maxTotal
-      const rank = Math.floor(((char.value - minTotal) / (maxTotal - minTotal)) * 10);
+      const rank = Math.floor(((char?.value - minTotal) / (maxTotal - minTotal)) * 10);
```

Comment:
Not at a computer, but what does `undefined - 3` evaluate as? The `char?.value` can be undefined and I'm thinking maybe doing `(char?.value ?? 0) ` is safer.

Reply:
We would have done a "continue" in the for loop above if char or char.value is null, so it shouldn't be undefined at that point, right?

Reply:
That's true so char should always have a value. Then there is no need to change it to `char?.value` right? Since that implies that char could be undefined.

Reply:
Agreed.  char, char.value, minTotal, and maxTotal all must have values at that point, or else we would have hit a continue.